concent_api/verifier/tasks.py

```diff
@@ -179,6 +183,33 @@ def blender_verification_order(
     for file_path in source_files_list + [source_package_path]:
         delete_file(file_path)
 
+    blender_output_file_name = generate_blender_output_file_name(scene_file)
```

Comment:
could this new block be extracted to a new function? (i.e. `upload_image()`)

Reply:
It is used only once now, so I don't see reason for this.

Reply:
just for improved clarity/readability ;)

Reply:
I would like to keep most of the verifier process in single place, if it is not a part that is currently reused, and then maybe take a better look on the whole task.